packages/db/src/core/cli/commands/link/index.ts

```diff
@@ -47,33 +87,95 @@ export async function cmd({ flags }: { config: AstroConfig; flags: Arguments })
 		process.exit(1);
 	}
 	const { data } = await response.json();
-	await mkdir(new URL('.', PROJECT_ID_FILE), { recursive: true });
-	await writeFile(PROJECT_ID_FILE, `${data.id}`);
-	console.info('Project linked.');
-}
 
-export async function promptProjectName(defaultName?: string): Promise<string> {
-	const { projectName } = await prompts({
-		type: 'text',
-		name: 'projectName',
-		message: 'Project ID',
-		initial: defaultName,
+	const { projectId } = await prompts({
+		type: 'autocomplete',
+		name: 'projectId',
+		message: 'What is your project name?',
+		limit: 5,
+		choices: data.map((p: any) => ({title: p.name, value: p.id})),
 	});
-	if (typeof projectName !== 'string') {
+	if (typeof projectId !== 'string') {
+		console.log('Canceled.')
 		process.exit(0);
 	}
-	return projectName;
+	return projectId;
+}
+
+export async function promptBegin(): Promise<void> {
+	// Get the current working directory relative to the user's home directory
+	const prettyCwd = process.cwd().replace(homedir(), '~');
```

Comment:
Please work on windows please work on windows please work on

Reply:
it *should...* right? `homedir()` is platform agnostic so I think this should work. We'll bug @itsMapleLeaf after merging to test :)